library/Icingadb/Compat/CompatObject.php

```diff
@@ -94,6 +94,10 @@ public function fetchCustomvars()
 
     public function __get($name)
     {
+        if ($name == 'action_url') {
+            $name = 'host_action_url';
+        }
+
```

Comment:
That's unfortunately the wrong solution. Please register `action_url` as a legacy column instead.